### PR TITLE
Package cleanups

### PR DESCRIPTION
**Pull request by albertodonato**
A few packaging cleanups:

- drop log dir creation since it's not used
- copy all `static` and `template` trees via `debian/install`

This also add an option to `buildpackage.sh` to skip running tests when building the package

### CODE REVIEW

File: buildpackage.sh [hunk 1]
```diff
@@ -67,8 +71,11 @@ fi
 # Get all the dependent go source.
 make -C $GOPATH/src/github.com/CanonicalLtd/candid deps
 
-go test github.com/CanonicalLtd/candid/...
-
+if [ -z "$NOTESTS" ]; then
+    go test github.com/CanonicalLtd/candid/...
+else
+    go get github.com/CanonicalLtd/candid/...
```
Comment (rogpeppe) [inline buildpackage.sh]: These two lines do quite different things - the second one installs binaries ("go install" would probably be a better way to spell it); the first just runs the tests without installing binaries.

I don't think you need to do anything if NOTESTS is set.

File: debian/preinst [hunk 2]
```diff
@@ -20,13 +20,11 @@ if ! getent passwd candid >/dev/null; then
     # Adding system user: candid.
     adduser \
       --system \
-          --disabled-login \
+      --disabled-login \
       --ingroup candid \
-      --gecos "identity server" \
+      --no-create-home \
+      --home /nonexistent \
+      --gecos "Canonical Identity Server" \
```
Comment (rogpeppe) [inline debian/preinst]: "Candid Identity Server" ?

File: buildpackage.sh (hunk 1 above)
Comment (albertodonato) [reply buildpackage.sh]: Currently this is used to fetch the source tree, but actually this makes me wonder how do we pull a specific version

File: debian/preinst (hunk 2 above)
Comment (albertodonato) [reply debian/preinst]: fixed

File: buildpackage.sh (hunk 1 above)
Comment (albertodonato) [reply buildpackage.sh]: @roaksoax how does this integrate with using uscan to fetch the source?

Comment (roaksoax) [reply buildpackage.sh]: debian/watch cannot yet be used to fetch a new source. buildpackage does it for you, however, it doesn't create a orig.tar.gz, whcih is require for uploading to the ubuntu archive.

Once i'm done cleaning the package, I'll move into improving the script.

**Comment by albertodonato** [comment]
Updated changes after rebasing on current master

**Pull request merged**